### ISSUE TITLE
Polling: vanilla js has code that runs before document has finished loading

### ISSUE THREAD

**Issue by villu164** [nikolalsvk/render_async#128]
https://github.com/renderedtext/render_async/blob/b33fd38533640f1a871ec4b89e2bc30ffef55832/app/views/render_async/_request_vanilla.js.erb#L119-L146

```js
<% if interval %>
...
  var container = document.getElementById('<%= container_id %>');

  // Register a polling stop event on the container
  container.addEventListener("async-stop", _clearRenderAsyncInterval)
```
This in turn makes vanilla js work only by doing the usual js-files at the bottom of the body

**Comment by nikolalsvk** [comment]
Got it, we should make this happen after the document is ready. Thanks for opening the issue, @villu164.

**Comment by nikolalsvk** [comment]
Hey, @villu164. I released a fix for this in the [2.1.8 version](https://github.com/renderedtext/render_async/releases/tag/2.1.8). Let me know if it is fixed for you.

Cheers 🍻
